draft-packages/form/KaizenDraft/Form/Primitives/FieldMessage/FieldMessage.tsx

```diff
@@ -1,26 +1,26 @@
-import classnames from "classnames"
 import * as React from "react"
-import exclamationIcon from "@kaizen/component-library/icons/exclamation.icon.svg"
-
 import { Icon, Paragraph } from "@kaizen/component-library"
+import exclamationWhiteIcon from "@kaizen/component-library/icons/exclamation-white.icon.svg"
+import classnames from "classnames"
 import styles from "./styles.scss"
 
-export type FieldMessageStatus = "default" | "success" | "error"
+export type FieldMessageStatus = "default" | "success" | "error" | "caution"
```

Comment:
```suggestion
export type FieldMessageStatus = "default" | "success" | "error" | "cautionary"
```

To match the language the designers use and also the pattern we seem to have for other components.
Ideally the others would change too, but that's pre-existing so it can be updated later.

Reply:
Good point, though I think it's fine here. We use `cautionary` in the *Mood* context combined with `assertive`, `informative`, ... which is slightly different to this.

Reply:
You mean to tell me that this isn't the same thing? 😂 Well now that just makes me look silly :P

Reply:
True, it'd be good to create a component health ticket for mood name updates so we can do it in one fell sweep :)